src/timer.h

```diff
@@ -97,6 +97,8 @@ class Timer {
   /// started.
   int time_;
 
+  int CompositionId;
```

Comment:
member variables should be named with `lower_case_and_underscores_` and have a trailing underscore for private vars